src/utils/data/setup-store.ts

```diff
@@ -62,14 +63,13 @@ export function setupStore(appState: AppState): { store: Store; local?: IndexedD
 
     const coordinator = new Coordinator({
       sources: [store, local, remote],
-      strategies: [remoteRequestStrategy, remoteStoreSync, localStoreSync, logger]
+      strategies: [remoteRequestStrategy, localStoreSync, logger]
     });
 
-    coordinator.activate();
```

Comment:
`pullIndexedDB` uses a flag to check whether is has been called before or not. The coordinator is only activated if it has not been called before.

Activating the coordinator in `pullIndexedDB` instead of `setupStore` helps us have better control because we can tell exactly when it has been activated (`pullIndexedDB` is a promise while `setupStore` should avoid having async calls because it's called on app start).